chart/templates/cleanup/cleanup-cronjob.yaml

```diff
@@ -105,10 +105,17 @@ spec:
               env:
                 {{- include "standard_airflow_environment" . | indent 12 }}
                 {{- include "container_extra_envs" (list . .Values.cleanup.env) | indent 12 }}
-              volumeMounts: {{- include "airflow_config_mount" . | nindent 16 }}
+              volumeMounts:
+                {{- include "airflow_config_mount" . | nindent 16 }}
+                {{- if .Values.volumeMounts }}
+                  {{- toYaml .Values.volumeMounts | nindent 16 }}
+                {{- end }}
               resources: {{- toYaml .Values.cleanup.resources | nindent 16 }}
           volumes:
             - name: config
               configMap:
                 name: {{ template "airflow_config" . }}
+            {{- if .Values.volumes }}
```

Comment:
I think here you could use 'with':
`
            {{- with .Values.volumes }}
            {{- toYaml . | nindent 12 }}
            {{- end }}
`

Reply:
yes I just copied this from other templates though (chart/files/pod-template-file.kubernetes-helm-yaml to be specific) and figured it would be best to use a consistent style. Can change if prefered though, just tell me.

Reply:
I didn't think about that. Lets just keep a consistent style 😄